### PR TITLE
Installing and configuring a payment gateway

### PR DESCRIPTION
**Pull request by krystalcode**
Page on how to install and configure a payment gateway via the UI + recipe on how to override a payment gateway's configuration in settings.php 
Related issue: https://github.com/drupalcommerce/commerce-docs/issues/107

### CODE REVIEW

**Comment by krystalcode** [comment]
Link to available-payment-gateways needs fixing.

**Comment by krystalcode** [comment]
Question, the original placeholder file was referring to make two examples:
> todo maybe two examples: like US and CA

Not sure what kind of examples depending on the country would these be?

File: v2/payments/installing-payment-gateway.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+=======================================
+
+While configuring your store you would probably want to install and configure a payment gateway provider module so that you can accept online payments. Currently available payment gateways for Commerce 2 are listed `here <available-payment-gateways.rst>`_.
+
+This tutorial will guide you through configuring a payment gateway provider using the Commerce PayPal module as an example.
+
+Install the Module
+------------------
+
+The first step is to install the module that provides integration with the desired payment gateway. In this case, Commerce PayPal. For information on how to install a module in Drupal 8 see `Installing Drupal 8 Modules <https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules>`_ and `Installing Modules from the Command Line <https://www.drupal.org/docs/8/extending-drupal-8/installing-modules-from-the-command-line>`_.
```
Comment (mglaman) [inline v2/payments/installing-payment-gateway.rst]: These do not explicitly detail to install via Composer. We're better off linking here http://pr-120-7kys47q-boljpk23hsit6.eu.platform.sh/v2/getting-started/composer.html

Comment (krystalcode) [inline v2/payments/installing-payment-gateway.rst]: Is it really a requirement to install via Composer? I downloaded and installed commerce_paypal using drush and it worked fine.

Comment (krystalcode) [inline v2/payments/installing-payment-gateway.rst]: Clarification, I installed Commerce modules via Composer, but the payment gateway provider module was installed via drush.

Comment (mglaman) [inline v2/payments/installing-payment-gateway.rst]: I didn't make the comment if it wasn't really dependency. Many payment gateway use an SDK and require Composer. PayPal does not due to the license of their SDK. However Braintree, Authnet and others use third party libraries

**Comment by mglaman** [comment]
Thanks @krystalcode !

**Pull request merged**